docs/index.md

```diff
@@ -1,5 +1,3 @@
-# Welcome to probatus documentation!
-
 <img src="img/logo_large.png" width=190 align="right">
 
 **Probatus** is a Python library that allows to analyse binary classification models as well as the data used to develop them.
```

Comment:
Since you're fixing docs here. Maybe we can fix the git clone link below?

Reply:
Even better. Idk if it's possible, maybe this index could be generated from the README.md when building? It's essentially the same info, but here it's totally outdated.

Reply:
Would be great! Do you perhaps have a proposal for that how to do that?

Reply:
I did some googling. It [may be not possible](https://github.com/mkdocs/mkdocs/issues/3152). So I guess I'd just keep the very basic and remove the outdated info, adding that the instructions on how to run/build are in the readme.md in the repo's root.